src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,53 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represent a deadline task that need to be done before a specific date/time.
+ */
+public class Deadline extends Task {
+    private LocalDateTime localDateTime;
+    private DateTimeFormatter dateTimeFormat = DateTimeFormatter.ofPattern("dd/MM/YYYY HHmm");
+    private String dateTime;
+
+    /**
+     * Initialise Deadline task with description of task and the date and time to be completed.
+     *
+     * @param description description of task
+     * @param dateTime    date and time to be completed
+     */
+    public Deadline(String description, String dateTime) {
```

Comment:
Maybe you could parse the date and time outside the constructor to make the method cleaner?